nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
```

Comment:
I agree with @tfoote regarding separate SetMap and LoadMap services. I'm interested in this functionality and plan to create the definitions and implement both in [map_server](https://github.com/ros-planning/navigation). If this sounds reasonable you can expect a PR on Wednesday.